changelogs/fragments/6199-archive-generate-checksum-in-chunks.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - archive - reduce ram usage by generating crc32 checksum over chunks
```

Comment:
```suggestion
  - archive - reduce RAM usage by generating CRC32 checksum over chunks (https://github.com/ansible-collections/community.general/pull/6274).
```

Reply:
I wasn't sure, unless you're compressing really big files it's not going to manifest as a bug but still impact the managed machine. Changed it to bugfix.

Reply:
Thanks for changing it to bugfix. Please note that there are some more suggestions, both to the format of the fragment and to some formatting (see the section on abbreviations in the Ansible docs style guide).